v0.3/tutorials/quickstart 1 button stopwatch.eve

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Quickstart + One Button Stopwatch
+---
+
+{% raw %}
+
+# Quickstart + One Button Stopwatch
+
+This guide expands on the quickstart `Clock` page and shows you how to turn the clock into a one button stopwatch. The following code is meant to replace all the code from the original clock.
+
+## Implement system time and a race timer
+
+```
+commit
+  [#time #system/timer resolution: 10]
+  [#race-timer #stopped #reset]
+```
+
+Stopwatches tend to measure seconds to two decimal points, so our resolution changes to 10ms (or one centisecond) to account for that. The other thing added here is a `#race-timer` record that lets us keep track of when the stopwatch is stopped or running or has been reset back to zero.
+
+
+## Draw clock face
+
+```
+search
+  [#app page: "Clock"]
+  view = [#qs-contents]
+  [#race-timer minute second millisecond]
+
+bind
+  view.children += [#svg/root sort: 1 viewBox: "0 0 100 100", width: "300px", children:
+    [#svg/circle #second-face sort: 1, cx: 50, cy: 50, r: 48, fill: "#fff", stroke: "#000", stroke-width: 0.5]
+    [#svg/circle #minute-face sort: 2, cx: 50, cy: 26, r: 12, fill: "#fff", stroke: "#000", stroke-width: 0.25]
+    [#minute-hand sort: 3, degrees: 6 * minute, length: 11, stroke: "#000", stroke-width: 0.5]
+    [#second-hand sort: 4, degrees: 6 * second + 0.006 * millisecond, length: 46, stroke: "#ff0000", stroke-width: 0.5]]
+```
+
+This block closely resembles its predecessor, but the search looks for `#race-timer` now instead of the system time since it's the race time that will determine the position of the hands.
+
+## One Button
+
+### Draw the button
+
+All we need to do is add one button to cycle between start, stop, and reset.
+
+```
+search
+  [#app page: "Clock"]
+  view = [#qs-contents]
+
+bind
+  view.children += [#ui/div sort: 2 children:
+    [#ui/div #qs-stopwatch-btn text: "Start/Stop/Reset"]]
+```
+
+### Pressing Start
+
+```
+search
+  [#time timestamp]
+  race-timer = [#race-timer #stopped #reset]
+  [#html/event/click element: [#qs-stopwatch-btn]]
+
+commit
+  [#start-time timestamp]
+  race-timer -= #stopped
+  race-timer -= #reset
+  race-timer += #running
+```
+
+If the button is pressed while the stopwatch is stopped and reset (thus at zero), this records the `#start-time` and changes the `#race-timer` from `#stopped` to `#running` and removes the `#reset` tag.
```

Comment:
Our code-style so far has been to write prose before the code block it describes.